apps/documentation/src/routes/documentation/rendering/static-site-generation.mdx

```diff
@@ -0,0 +1,39 @@
+import MetaTags from '@/components/MetaTags'
+
+<MetaTags
+  title="Tuono - Static site Generation"
+  canonical="https://tuono.dev/documentation/rendering/static-site-generation"
+/>
+
+import Breadcrumbs from '@/components/Breadcrumbs'
+
+<Breadcrumbs breadcrumbs={[{ label: 'SSG' }]} />
+
+# Static Site Generation
+
+## Overview
+
+> 🚧 This feature is experimental and still misses few case scenarios. It will be stabilized soon.
+
+Tuono is a progressive framework which means that fills all the development needs from a basic
+app with no server capabilities to a full server side rendered app with dynamic data.
+
+If a page uses Static Generation, the page HTML is generated at build time.
+This HTML will then be reused on each request and it can be cached by a CDN.
```

Comment:
```suggestion
If a page uses Static Generation, its HTML is generated at build time and reused on each request. 
It can also be cached by a CDN.
```